standup/README.md

```diff
@@ -0,0 +1,44 @@
+# Firestore Standup collection data model
+
+```json
+{
+  "id": "String",          // Unique identifier for the document
+  "type": "String",        // Type of the document, e.g., "task progress" or "user progress"
+  "userId": "String",      // User ID (applicable for both task progress and user progress)
+  "taskId": "String",      // Task ID (applicable for task only)
```

Comment:
we can just keep it as id because the type is already present. for progress the user id will be empty for standup task id will be empty
it will help in data consistency. 
Also, task id won't exist for standup.
So to get the user associated with the task we had planned to create a mapping collection right?

Reply:
`we can just keep it as id because the type is already present. for progress the user id will be empty for standup task id will be empty.it will help in data consistency.`
For task we would need both task id and user id to track which user is working on the task. For Standup updates task id wont be needed.

Reply:
`So to get the user associated with the task we had planned to create a mapping collection right`
for a single task there can be multiple users working on it simultaneously. if we store the task - user association in a different collection it will not be possible to figure out just from task progress documents that which task document belong for which user. Let me know your thoughts

Reply:
`Also, task id won't exist for standup.`
Agreed.